.github/workflows/integration.yaml

```
@@ -21,14 +21,21 @@ on:
   push:
     branches:
       - main
+      - 'maint-**'
```

```suggestion
      - 'maint-*'
```
Probably not an issue but I'd say `**` only makes sense for paths